source/voice_control/s3-box-customize.markdown

```diff
@@ -0,0 +1,167 @@
+   - Depending on your environment, the installation process can take a while.
+![ESP32-S3-BOX-3 config file](/images/assist/esp32-s3-config-05.png)
+   
+5. Once the installation is complete, you can see the new image on the ESP32-S3-BOX-3.
+   - Now, speak a command to test the new setting. For example, *OK Nabu turn off the living room lights*.
+
+## Option 2: Using your own images
+
+There are 2 parts to this:
+
+- [Preparing your own images](#to-prepare-your-own-images)
+- [Adding your images to the configuration](#to-add-your-images-to-the-configuration)
+
+### About the images
+
+Here's what you need to know so that the images can be displayed properly on the ESP32-S3-BOX-3 screen.
+
+#### Illustrating 6 different states
+
+There are 6 different states to illustrate:
+
+- loading, idle, listening, thinking, replying, error
+
+![ESP32-S3-BOX-3 config file](/images/assist/s3-box-status-images.png)
+
+#### Using light and dark image background
+
+To make it easier to recognize a state change when you look at your screen, in your image, you could use 2 different background colors:
+
+- For loading and idle: use a dark background
+- For the listening, thinking, replying: use a light background
+- For error: As you like
+  
+If your images have transparency, you can define the background color in the configuration. There is a step in the procedure below showing you how you can change this.
+
+#### Image dimensions and file format
+
+- **Dimensions**: The screen is 320 x 240 pixels. If the image you provide is not in a 4:3 ratio, the background color will be shown in the remaining area.
+- **File format**: png or jpeg
+
+### To prepare your own images
+
+1. Create your own images according the specifications defined [above](#about-the-images).
+2. Copy all 6 images into a folder. For example: `voice_assistant_pictures`.
+3. Make sure you have [access to your configuration files](/common-tasks/os/#configuring-access-to-files).
+   - [Install the Samba add-on](/common-tasks/os/#installing-and-using-the-samba-add-on).
+   - This allows you to copy paste multiple files at once.
+4. Copy your image folder into the configuration folder:
+   - Open the `config` folder and open the `ESPHome` folder.
+   - Copy your image folder in there.
+   ![ESP32-S3-BOX-3 config file](/images/assist/s32-s3-add-image-folder.png)
+
+### To add your images to the configuration
+
+1. In Home Assistant, go to [**Settings** > **Add-ons** > **ESPHome**](https://my.home-assistant.io/redirect/supervisor_addon/?addon=5c53de3b_esphome), and **Open Web UI**.
+2. On the **ESP32-S3-BOX-3** add-on, select edit.
+   - **Result**: An editor opens, showing the configuration file.
+   ![ESP32-S3-BOX-3 config file](/images/assist/esp32-adopt-s3-01.png)
+
+3. To add your images, add the following lines into the `substitution` block.
+
+   ```yaml
+   substitution:
+     loading_illustration_file: /config/esphome/voice_assistant_pictures/loading_320_240.png
```

Comment:
Make all these paths relative to the folder that contains the ESPHome configuration. 
```suggestion
     loading_illustration_file: ./voice_assistant_pictures/loading_320_240.png
```